src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java

```diff
@@ -315,7 +314,7 @@ public boolean saveMainConfig() {
             this.MVPConfig.save(new File(this.getDataFolder(), "config.yml"));
             return true;
         } catch (IOException e) {
-            this.log(Level.SEVERE, "Failed to save Portals config.yml.");
+            Logging.info("Failed to save Portals config.yml.");
```

Comment:
Was this change intentional?

Reply:
ahh probably a mistake, had to change so many haha